pkg/kubernetes/scripts/tags.js

```diff
@@ -73,8 +73,10 @@
         });
         var tags = [ ];
         angular.forEach(names, function(name) {
-            if (name in already)
+            if (name in already) {
+                already[name].importPolicy = { "insecure": insecure };
                 tags.push(already[name]);
+            }
             else
                 tags.push({ name: name });
```

Comment:
Why does this only apply to tags that already exist? What about newly created tags here on line 81?